### DISCUSSION TITLE
Desktop App

### DISCUSSION THREAD

**Discussion by trymeouteh** [bmaroti9/Overmorrow#25] [Ideas]
Please release this app on Windows, MacOS and Linux and allow the app to run in the desktop tray.

**Comment by bmaroti9** [comment]
Hi! I have already thought of this, and it seems like a good idea, but very complicated. I only tried Linux building, and the locating and stuff were all messed up, not even talking about the fact that it's very hacky to make a flutter app run on linux. As for windows and macOs, i have no experience, but i think MacOs would be even more difficult, because you need Xcode and a mac. Overall not a bad or ignored idea, just very low priority. i wan't to make the phone app good first.
